step-ca/README.mdx

```diff
@@ -88,3 +88,22 @@ For strong protection of your CA signing keys, we've built [`step-ca` integratio
 ### Other Integrations
 
 `step-ca` plays well with Kubernetes cert-manager and Envoy Secret Discovery Service. For more information, see [Integrations](./integrations.mdx).
+
+## Limitations
+
+While we think `step-ca` is the best open source, scalable online Certificate Authority on the internet, every piece of software comes with limitations and tradeoffs.
+
+`step-ca` is designed to favor a simple deployment of a scalable two-tiered X.509 PKI, with one Root CA and one Intermediate CA that issues end-entity certificates with passive revocation. Its SSH support is favors having two single-tier SSH CAs—a single SSH Host CA, and a single SSH User CA.
```

Comment:
For SSH, maybe drop the single-tier part. You can only have a single-tier with SSH (unless using X509 certs to auth in SSH)
```suggestion
`step-ca` is designed to favor a simple deployment of a scalable two-tiered X.509 PKI, with one Root CA and one Intermediate CA that issues end-entity certificates with passive revocation. Its SSH support favors having two single-tier SSH CAs—a single SSH Host CA, and a single SSH User CA.
```

Reply:
```suggestion
`step-ca` is designed to favor a simple deployment of a scalable two-tiered X.509 PKI, with one Root CA and one Intermediate CA that issues end-entity certificates with passive revocation.
```

Yeah I'll just drop the whole SSH sentence.